elements/xml_form_elements.module

```diff
@@ -300,6 +300,17 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
   $new_tab = clone $tab;
   $new_tab->eachDecendant(function($element) {
     $element->default_value = $element->getOriginalDefaultValue();
+                            if ($element->controls['#type'] == 'tags') {
```

Comment:
@willtp87 I had to add the whitespace here because it was making some of the travis builds fail. I can remove the whitespace but it requested it be added and it was the only thing that made them pass.